examples/Huggingface_Transformers/Image_classification_docker/HF-only/scripts/torchserve_vitxxsmall_handler.py

```diff
@@ -0,0 +1,171 @@
+'''
+References: 
+(i) https://github.com/pytorch/serve/blob/master/docs/custom_service.md
+(ii) https://github.com/pytorch/serve/blob/master/ts/torch_handler/base_handler.py 
+(iii) https://github.com/pytorch/serve/blob/master/ts/torch_handler/text_handler.py
+(iv) https://github.com/cceyda/lit-NER/blob/master/lit_ner/serve_pretrained.py
+
+Reference that implements handler class without inheriting the `BaseHandler`:
+(i) https://github.com/aws-samples/amazon-sagemaker-endpoint-deployment-of-siamese-network-with-torchserve/blob/main/deployment/handler.py 
+
+
+Batching 🤗 model & Torchserve inputs in the handler: https://youtu.be/M6adb1j2jPI
+https://github.com/pytorch/serve/issues/1783
+
+'''
+
+import os
+import torch
+import base64
+import io
+from PIL import Image
+
+from ts.torch_handler.base_handler import BaseHandler
+from transformers import *
+
+
+#Source: https://huggingface.co/docs/transformers/v4.21.2/en/main_classes/pipelines#transformers.pipeline.task
+pipeline_module_map = {
```

Comment:
This is cool but it doesn't seem like its quite used. My expectation was your PR would provide an easy way to support all existing pipelines but that doesn't quite seem to be the case.

Users still need to
1. Pick a task
2. Branch on pipeline type
3. Figure out the right way to encode/decode data
4. Figure out how to work with a specific pipeline

I think it's possible to automate this completely on behalf of end users

Reply:
Yeah currently that seems to be the case, but note that I forgot to remove the redundancy of the dictionary, i.e, the code can be this too without any change in the functionality:
```
from transformers import pipeline

pipeline_supported_tasks= [
    "audio-classification",
    "automatic-speech-recognition",
    "conversational",
    "feature-extraction",
    "fill-mask",
    "image-classification",
    "question-answering",
    "table-question-answering,
    "text2text-generation",
    "text-classification",
    "sentiment-analysis",
    "text-generation",
    "token-classification",
    "ner",
    "translation",
    "translation_xx_to_yy",
    "summarization",
    "zero-shot-classification"
]

task="image-classification"
framework="pt"

try:
    assert task in pipeline_supported_tasks
except:
    print("Enter a task supported by 🤗 pipeline")
    exit(0)
```

Users still need to:
1. Pick a task
3. Figure out the right way to encode/decode data
4. Figure out how to work with a specific pipeline

It seems possible to automate the part 3 and 4 by going through some documentations, experimenting and time